repository-data/application/src/main/resources/hcm-content/content/documents/corporate-website/faceted-search-by-date.yaml

```diff
@@ -1,29 +1,34 @@
 ---
-/content/documents/corporate-website/facet:
+/content/documents/corporate-website/faceted-search-by-date:
   hippo:docbase: 12345678-1234-1234-1234-123456789abc
   hippofacnav:facetnodenames:
+  - category
   - assuredStatus
   - document-type
   - year${sortby:'facetvalue', sortorder:'descending'}
   - month${after:'year', sortby:'facetvalue', sortorder:'descending',hide:'year'}
   - publicationStatus
-  - category
   - information-type
   - geographical-coverage
   - geographical-granularity
   - publishedBy
   - reportingLevel
   hippofacnav:facets:
+  - common:FullTaxonomy
   - nationalindicatorlibrary:assuredStatus
   - common:FacetType
   - publicationsystem:NominalDate$year
   - publicationsystem:NominalDate$month
   - publicationsystem:PubliclyAccessible
-  - common:FullTaxonomy
   - publicationsystem:InformationType
   - publicationsystem:GeographicCoverage
   - publicationsystem:Granularity
   - nationalindicatorlibrary:publishedBy
   - nationalindicatorlibrary:reportingLevel
   hippofacnav:limit: 100
+  hippofacnav:sortby:
+  - common:orderedSearchDate
+  - nationalindicatorlibrary:assuranceDate
+  - common:searchRank
+  - hippostdpubwf:lastModificationDate
```

Comment:
Perhaps you should also specify the sortorder, as it defaults to ascending if not specified, but the component was using descending. See https://xmdocumentation.bloomreach.com/library/concepts/faceted-navigation/configure-sorting-and-limiting-of-result-sets.html

Reply:
Good spot. I will add that.